ci/build_docs.sh

```diff
@@ -5,6 +5,9 @@ set -euo pipefail
 
 export RAPIDS_VERSION_NUMBER="$(rapids-generate-version)"
 
+export RAPIDS_VERSION="$(cat VERSION)"
+export RAPIDS_VERSION_MAJOR_MINOR="$(sed -E -e 's/^([0-9]{2})\.([0-9]{2})\.([0-9]{2})/\1.\2/' VERSION)"
```

Comment:
Since this little sed script is used repeatedly, can we make it a utility function so it is reusable and documented?

Reply:
Opened https://github.com/rapidsai/gha-tools/pull/97.

Reply:
If we rely on this from `gha-tools`, local environments without `gha-tools` installed will be unable to get the version information. I think this could be problematic for local builds if we call this anywhere except in CI scripts.

Reply:
Is there any other repository with shared tools that's accessible for local builds? Otherwise, we're just copying these utilities everywhere, and then we're right back where we started.

Also, when running Doxygen locally, users have the option of using the CMake `doxygen` target.

Reply:
Generally no, there is no set of shell scripts that all developers will have locally.

Reply:
What if the rapids_version scripts lived in rapids-cmake instead?

Or could we use rapids-cmake / cpm to get it from gha-tools?